src/System.Security.Cryptography.Xml/tests/EncryptedXmlTest.cs

```diff
@@ -332,6 +332,124 @@ public void DecryptEncryptedKey_Null()
             Assert.Throws<ArgumentNullException>(() => ex.DecryptEncryptedKey(null));
         }
 
+        [Fact]
+        public void EncryptKey_TripleDES()
+        {
+            using (TripleDES tripleDES = TripleDES.Create())
+            {
+                byte[] key = Encoding.ASCII.GetBytes("123456781234567812345678");
+
+                byte[] encryptedKey = EncryptedXml.EncryptKey(key, tripleDES);
+
+                Assert.NotNull(encryptedKey);
+                Assert.Equal(key, EncryptedXml.DecryptKey(encryptedKey, tripleDES));
+            }
+        }
+
+
+        [Fact]
+        public void EncryptKey_AES()
+        {
+            using (Aes aes = Aes.Create())
+            {
+                byte[] key = Encoding.ASCII.GetBytes("123456781234567812345678");
+
+                byte[] encryptedKey = EncryptedXml.EncryptKey(key, aes);
+
+                Assert.NotNull(encryptedKey);
+                Assert.Equal(key, EncryptedXml.DecryptKey(encryptedKey, aes));
+            }
+        }
+
+        [Fact]
+        public void EncryptKey_AES8Bytes()
+        {
+            using (Aes aes = Aes.Create())
+            {
+                byte[] key = Encoding.ASCII.GetBytes("12345678");
+
+                byte[] encryptedKey = EncryptedXml.EncryptKey(key, aes);
+                
+                Assert.NotNull(encryptedKey);
+                Assert.Equal(key, EncryptedXml.DecryptKey(encryptedKey, aes));
+            }
+        }
+
+        [Fact]
+        public void EncryptKey_AESNotDivisibleBy8()
+        {
+            using (Aes aes = Aes.Create())
+            {
+                byte[] key = Encoding.ASCII.GetBytes("1234567");
+
+                Assert.Throws<CryptographicException>(() => EncryptedXml.EncryptKey(key, aes));
+            }
+        }
+
+        [Fact]
+        public void DecryptKey_TripleDESWrongKeySize()
+        {
+            using (TripleDES tripleDES = TripleDES.Create())
+            {
+                byte[] key = Encoding.ASCII.GetBytes("123");
+
+                Assert.Throws<CryptographicException>(() => EncryptedXml.DecryptKey(key, tripleDES));
+            }
+        }
+
+        [Fact]
+        public void DecryptKey_TripleDESCorruptedKey()
+        {
+            using (TripleDES tripleDES = TripleDES.Create())
+            {
+                byte[] key = Encoding.ASCII.GetBytes("123456781234567812345678");
+
+                byte[] encryptedKey = EncryptedXml.EncryptKey(key, tripleDES);
+                encryptedKey[0] = (byte)(encryptedKey[0] > 0 ? encryptedKey[0] - 1 : 1);
```

Comment:
You seem to have changed the other two instances of this to ^= 0xFF, but not this one.  Was that a mistake, or is this one special?